Mortgage-loan-calculator/src/app/admin-login/helpers/http.interceptor.ts

```diff
@@ -1,27 +0,0 @@
-import { Injectable } from '@angular/core';
-import { HttpEvent, HttpInterceptor, HttpHandler, HttpRequest, HTTP_INTERCEPTORS } from '@angular/common/http';
-import { Observable } from 'rxjs';
-import {StorageService} from "../services/storage.service";
-
-const TOKEN_HEADER_KEY = 'Authorization';
-@Injectable()
-export class HttpRequestInterceptor implements HttpInterceptor {
-  constructor(private storage: StorageService) { }
-  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    const token = this.storage.getToken();
-    if (token != null) {
-      req = req.clone({ headers: req.headers.set(TOKEN_HEADER_KEY, 'Bearer ' + token) });
-    }
-    else{
-      req = req.clone({
-        withCredentials: true,
```

Comment:
Is backend using any cookies so that they should be sent? Maybe `withCredentials` flag is not needed at all?